datahub-web-react/src/app/shared/share/items/EmailMenuItem.tsx

```diff
@@ -2,7 +2,7 @@ import React, { useState } from 'react';
 import styled from 'styled-components';
 import { Tooltip } from 'antd';
 import { CheckOutlined, MailOutlined } from '@ant-design/icons';
-import EmailShare from 'react-email-share-link';
+import qs from 'query-string';
```

Comment:
Yeah this component doesn't even work for me on our datahub demo site... seems like you have parity with the react-email-share-link functionality though.

Reply:
Ah, it's probably because I didn't set my mail client. When I did mac mail, it worked. So I think this is all good.

Reply:
Yea it should be the exact same behaviour since the code was copied as-is. Glad to hear it works!